easy-ui-react/src/utilities/storybook.tsx

```diff
@@ -38,3 +40,27 @@ function getTokensControl(tokenAliases: string[]) {
     tokenAliases.reduce((o, alias) => ({ ...o, [alias]: alias }), {}),
   );
 }
+
+const buttonStoryStyles = {
+  display: "flex",
+  alignItems: "center",
+  gap: "12px",
+  flexWrap: "wrap",
+} as React.CSSProperties;
+
+const buttonStoryOnDarkBackgroundStyles = {
+  backgroundColor: "var(--ezui-color-blue-800)",
+  padding: "12px",
+};
+
+export const ButtonStoryDecorator: Decorator = (Story) => (
+  <div style={buttonStoryStyles}>
+    <Story />
+  </div>
+);
+
+export const ButtonStoryOnDarkBackgroundDecorator: Decorator = (Story) => (
+  <div style={{ ...buttonStoryOnDarkBackgroundStyles, ...buttonStoryStyles }}>
+    <Story />
+  </div>
+);
```

Comment:
**suggestion:** doesn't necessarily have to be done now, can be a follow up, but as you pointed out on a previous PR, we could probably reuse these decorators for other inline components. what are your thoughts on renaming to `inline` and `Inline`?